### PR TITLE
Update Getting started - Setup environment.md

### PR DESCRIPTION
**Pull request by SeSeicht**
# Change

just set the correct name to the description

### CODE REVIEW

File: docs/wiki/Getting started - Setup environment.md [hunk 1]
```diff
@@ -291,7 +291,7 @@ The primary pipeline variable file `.azuredevops/pipelineVariables/global.variab
 | - | - | - |
 | `bicepRegistryName` | `'adpsxxazacrx001'` | The container registry to publish Bicep templates to. <p> **NOTE:** Must be globally unique |
 | `bicepRegistryRGName` | `'artifacts-rg'` | The resource group of the container registry to publish Bicep templates to. Is used to create a new container registry if not yet existing |
-| `bicepRegistryRGName` | `'artifacts-rg'` | The location of the resource group of the container registry to publish Bicep templates to. Is used to create a new resource group if not yet existing |
+| `bicepRegistryRgLocation` | `'West Europe'` | The location of the resource group of the container registry to publish Bicep templates to. Is used to create a new resource group if not yet existing |
```
[the remark this answers is not in the document]
Comment (SeSeicht) [reply docs/wiki/Getting started - Setup environment.md:303]: done

File: docs/wiki/Getting started - Setup environment.md [hunk 2]
```diff
@@ -2,12 +2,21 @@ The repository is built so that you can create your own 1:1 instance and be able
...
 This requires several steps:
 
-1. [Configure your Azure environment](#1-configure-your-azure-environment)
-1. [Fork/clone the repository into your DevOps environment](#2-forkclone-the-repository-into-your-devops-environment)
-1. [Configure the CI environment](#3-configure-the-ci-environment)
-1. [Deploy dependencies](#4-deploy-dependencies)
-1. [Update module parameter files](#5-update-module-parameter-files)
-1. [(Optional) Convert library to ARM](#6-optional-convert-library-to-arm)
+- [1. Configure your Azure environment](#1-configure-your-azure-environment)
+- [2. Fork/clone the repository into your DevOps environment](#2-forkclone-the-repository-into-your-devops-environment)
```
Comment (MrMCake) [inline docs/wiki/Getting started - Setup environment.md:6]: Something tells me you're using a markdown extension. If you can, please undo the number updates. It better to have markdown take care of it using native capabilities

Comment (MrMCake) [reply docs/wiki/Getting started - Setup environment.md:6]: Not sure though if it works with nested lists

Comment (SeSeicht) [reply docs/wiki/Getting started - Setup environment.md]: indeed. should now be removed, thanks!

**Review by MrMCake — APPROVED** [review]

**Pull request merged**